gitlab-runner-17.1.yaml

```diff
@@ -3,12 +3,15 @@
 # however it will be easy to fix
 #nolint:git-checkout-must-use-github-updates
 package:
-  name: gitlab-runner
+  name: gitlab-runner-17.1
   version: 17.1.0
-  epoch: 1
+  epoch: 2
```

Comment:
you can reset the epoch to zero in case you rename the package to something else